docs/_docs/04_community/contributing.md

```diff
@@ -150,16 +149,50 @@ to include the following:
    test output so we can verify that everything is working.
 1. Any notes on backwards incompatibility or downtime.
 
+#### Validate the Pull Request for Azure Platform
+
+After [creating a pull request](https://help.github.com/articles/creating-a-pull-request/), please follow the below guideline for [Azure Platform](https://azure.com)
+
+##### To run the CI Pipeline on forked repo
+
+1. Run the following command on your preffered Terminal and copy the output
+
+```bash
+az ad sp create-for-rbac --name "terratest-az-cli" --role contributor --sdk-auth
+```

```

Comment:
If you indent the code block by 4 spaces, it'll appear properly as a "sub item" of the list.

```suggestion
    ```bash
    az ad sp create-for-rbac --name "terratest-az-cli" --role contributor --sdk-auth
    ```

```

Reply:
cool, thanks!..